docs/embedded/tutorials/create-containertype.md

```diff
@@ -0,0 +1,133 @@
+
+
+
+## Set up an Azure Subscription Id, Resource Group, and Region
+
+### 1. Set up Azure Subscription Id
+Login to Azure Portal with the admin credentials of your tenant
+
+
+(TODO: add the extra existing Sub id)
+
+Navigate to [Subscriptions](https://portal.azure.com/#view/Microsoft_Azure_Billing/SubscriptionsBladeV2)
+![alt text](./regct-image/azuresubscription.png)
+
+If you do not have an existing subscription set-up, click add
+![alt text](./regct-image/suboption.png)
+
+Choose a subscription plan. You will be prompted to add in credit card information that will be billed when you create File Storage Containers. This will set you up with an Azure Subscription Id.
+
+### 2. Set up a Resource Group and Region
+Navigate to [Resource Groups](https://portal.azure.com/#view/HubsExtension/BrowseResourceGroups) in Azure Portal.
+![alt text](./regct-image/resource-group.png)
+
+Click "Create" to create a resource group and name it something appropiate to your application. Then click Review + Create
+![alt text](./regct-image/c-rg.png)
+
+Make sure that:
+* The appropiate Azure Subcription is selected for the resource group
+* The appropiate physical region is selected. For ex. (US) East US
+
+>> Note: The Region Group does NOT require your Containers to live in that specified region. Your content is not restricted to the specified region
+
+Region refers to where the CT is created  for example “US-East”. This does NOT affect geographically where containers can live and does not affect geographically where applications are installed. Also, regions do not place limits on where SharePoint Embedded Applications can be used.
+
+Click Review + Create and your validation should pass if the inputs are correct.
+
+Now you have all the necessary resources to create a Standard Container Type which will defines the billing relationship between your Containers and Azure SubscriptionId.
+
+<br></br>
+
+## Create your Container Type
+
+### 3. Use PowerShell to create a new Container Type for your Application
+To learn more about the role of Container Types in SharePoint Embedded please refer to the [ContainerTypes](../concepts/app-concepts/containertypes.md) and [App Archictecture](../concepts/app-concepts/app-architecture.md) articles. To learn more about trial containers which are limited, but don't incur costs during the development of your application, see [Developer Admin](../concepts/admin-exp/dev-admin.md).
+
+You need to install and import the latest **Microsoft.Online.SharePoint.PowerShell** module.
+
+```powershell
+Install-Module "Microsoft.Online.SharePoint.PowerShell" # if you don't have it already
+# OR
+Update-Module "Microsoft.Online.SharePoint.PowerShell" # if you have it already
+```
+
+Have the following information handy from [Azure Portal](portal.azure.com):
```

Comment:
Invalid link (see earlier comment).

Please see earlier comment about line breaks for headings & paragraphs